### ISSUE TITLE
Merge tags block positioning issues

### ISSUE THREAD

**Issue by ketys-from-meiro** [zalify/easy-email#258]
Hi guys, I'm trying to implement our own merge tags picker into the easy email editor but I've faced some problems with positioning. We have two-levels picker for better orientation in it, so imajin it as a data source -> attribute where attribute === tag.

When I for example insert the image block and want to choose some tag for its url, it opens above the input field and is cropped at the top if I'm not scrolled down with the whole page:
<img width="387" alt="Screenshot 2023-03-01 at 11 53 20" src="https://user-images.githubusercontent.com/68279944/222123826-c0cce381-9f54-4b93-8116-7125bd551284.png">

Additionally when I start scrolling in the sidebar, it's still there fixed in the upper part of the easy email's sidebar.
<img width="390" alt="Screenshot 2023-03-01 at 11 53 35" src="https://user-images.githubusercontent.com/68279944/222123817-55e1dd58-a830-4621-a9e6-d1aee9923b80.png">

Last thing, when I scroll down a bit so it opens properly, then choose some data source with low number of attributes, the window can fit above the input field and it creates some weird jumping effect where sometimes you have the dropdown below, sometimes above.
<img width="385" alt="Screenshot 2023-03-01 at 11 53 41" src="https://user-images.githubusercontent.com/68279944/222123808-7e712acb-1b14-4044-bcab-5150d34a0d7b.png">

Do you have some suggestions what can I do with that? Is it something which could be potentially fixed in your repo? Thanks a lot!

**Comment by m-Ryan** [comment]
![image](https://user-images.githubusercontent.com/20615214/222393870-0182fa5d-c4ad-4169-a7fe-80704259d76b.png)

Try `renderMergeTagContent`

**Comment by ketys-from-meiro** [comment]
@m-Ryan yeah that's what I used, don't have problem it doesn't work at all, but the positioning of the merge tags block in sidebar is kinda clunky and sometimes bugged

**Comment by m-Ryan** [comment]
I don't quite understand your meaning, more information is needed.

**Comment by ketys-from-meiro** [comment]
@m-Ryan just try to place inside `renderMergeTagContent` some tall div and it will be cut in the upper edge of the editor area
